backend/src/main/java/pl/krkteam/authentication/PlayerHolder.java

```diff
@@ -0,0 +1,33 @@
+package pl.krkteam.authentication;
+
+import pl.krkteam.player.Player;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class PlayerHolder {
+
+    private Set<Player> players = new HashSet<>();;
+
+    public boolean addPlayer(Player player) {
+        if (player == null) {
+            throw new IllegalArgumentException("Player cannot be null");
+        }
+        System.out.println("Player " + player.getName() + " has been added to PlayerHolder");
```

Comment:
please consider using external logger

Reply:
Refactor with logger will be done at the end of this iteration.